e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/PipelineLibraryContentAO.java

```diff
@@ -132,6 +138,12 @@ public PipelineLibraryContentAO assertRunButtonIsDisplayed() {
         return this;
     }
 
+    public PipelineRunFormAO clickRunButton() {
+        sleep(5, SECONDS);
+        $$(tagName("button")).find(text("Run")).shouldBe(visible).click();
+        return new PipelineRunFormAO();
+    }
```

Comment:
There are ready methods for this. Use clickOnPipeline(pipeline).firstVersion().runPipeline() instead of this

Reply:
Done